### PR TITLE
added additional fields to config files

### PR DESCRIPTION
**Pull request by TannerGilbert**

### CODE REVIEW

**Comment by TannerGilbert** [comment]
Related to keptn-sandbox/artifacthub#1

File: jira/0.8.5/artifacthub-pkg.yml [hunk 1]
```diff
@@ -3,8 +3,9 @@
...
...dOhVfhbGmvqRmz1MrmHWwEMYRyFrAAeldDB1BLPI2GBhfLIbNaPWeQDWTsU40UtsiLqX2I6+abX025effyr8U9yhYusoKShgBDpC6d8rx8uUiIEjmAs0cwCgStjLWVyDVjROS/ium8V/tXomS/ium8V/tXotKfOZesiVIiLBIIiIAo9jlgY1MHcC80Rzmc/hM6R6QFIUVlKpKlNTjmiFSCnFxeTKFRWRjTicJiZqezZTpcw6GvPKD8l3oPNrVe1lJJE7MlY5jxscLdI5RzhewwuLp4mN4Z71vX7Xev4ebxGHnQdpZbnufzsPUxIb8Oh3v9m5XAqixK+PRb3+zcrdXD0308ft95HV0X0T8fZBERcc6QREQBERAEREAREQHDURB7HMOgPaW33iyp3COR2pb8WqYZANQkDo3brtzgfQroRSjNxyIuKeZnybJlhZuqBrudssf9RBX7FkxwsdcDG87pY/6SStBIrOWkY5NFQ4DyPuuHVs7Q0f5cNyTzGRwFuhvSFZ+CcFw00Yip42xxt1NbtO1xJ0uJ5TpXfRVym5ZklFLIIiKJk+SNCok5J8Kf7X7139ivhFOE3HIjKKeZVOIOINdSVrJ5+A4NrJAcyQuN3NsNBaFayIsSk5O7MpJKyCqnKBiDXVla6eDgODcyNoz5C03aLHQGlWsiRk4u6DSasyioMlWFA5pPuWwc0n312oEHwFeqIsym5ZmIxSyCIigSCIiA4KqnZIx0cjWvjeCHNcAWuadYIOsKrcZskVyX4PkDQf8mUmw5myaTbmcDvVsopRk45GGk8zN9XiJhSM2dSSnnjzZAfsErgixPwk42FFU9MZb6XWC0uis5d9hDk0UTgfJRhCUgzmOnZtuRI/oY0263BWpitijSUDCIGkyOHdyvsXu5r6mt/0iw6dKkSKEqkpZklFIgWVHFSqrxT+5uC95Mudwji3v+DtazTfvSoH+ybCn+1+9d/Yr5RZjVlFWRhwTdzwcSsFyUtFDTzEcJEHB2Zpbpe5wsbDYQi95FW9ruSsEREMhERAEREAXXqKaORubIxj28j2hw6iiK7DdNHxIVejkV/gyFrcKRhjWtHd6GgAf4Z2BWQiLoaY6WH2rjI0tHdHL7nwQREXJOgEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREB//Z
```
Comment (jetzlstorfer) [inline jira/0.8.5/artifacthub-pkg.yml:8]: is this correct?

**Review by jetzlstorfer — APPROVED** [review]
one logo URL seems a bit suspicious, but looks fine in general

File: jira/0.8.5/artifacthub-pkg.yml (hunk 1 above)
Comment (TannerGilbert) [reply jira/0.8.5/artifacthub-pkg.yml]: Wasn't sure if it would work with Artifacthub. Therefore I just changed it.

**Review by jetzlstorfer — APPROVED** [review]
lgtm

**Pull request merged**